canvas_modules/common-canvas/src/common-canvas/svg-canvas-d3.js

```diff
@@ -147,8 +160,11 @@ export default class SVGCanvasD3 {
 
 					} else if (CanvasUtils.isCmndCtrlPressed(d3Event) && d3Event.keyCode === V_KEY && actions.pasteFromClipboard) {
 						CanvasUtils.stopPropagationAndPreventDefault(d3Event);
-						this.canvasController.keyboardActionHandler("paste");
-
+						if (this.mousePos) { this.mousePos = this.renderer.convertPageCoordsToCanvasCoords(this.mousePos.x, this.mousePos.y);
```

Comment:
The line of code inside the curly bracket should be o the next line. Again, lint should be telling you that is a format error.  Please get lint set up correclty.